packages/ember-toucan-core/src/-private/components/error.hbs

```diff
@@ -3,21 +3,22 @@
     {{if this.hasMoreThanOneError 'items-start' 'items-center'}}"
   ...attributes
 >
-  {{! TODO: https://github.com/CrowdStrike/ember-toucan-core/issues/49 - Placeholder icon }}
   <svg
-    class="mr-1 h-3 w-3"
-    xmlns="http://www.w3.org/2000/svg"
-    width="24"
-    height="24"
-    stroke="currentColor"
-    viewBox="0 0 24 24"
+    aria-hidden="true"
```

Comment:
Should `aria-hidden` true be on all icons? TIL...

Reply:
Yeah! I think it's generally a recommended practice to remove it from the Accessibility Tree as they're only presentational additions.  We have some existing code doing this as well.  [This stack overflow](https://stackoverflow.com/questions/61048356/why-do-we-use-aria-hidden-with-icons) explains it a bit better (they use `<i` instead)